lib/db/seed.js

```diff
@@ -0,0 +1,349 @@
+const { db } = require("./db");
+const { codes, metadata, code_group_mappings } = require("../../db-schema");
+
+const SENSITIVITY_TAGGING_RULES = [
```

Comment:
I think we don't need to preserve this form and we can directly seed the database.